packages/gatsby-plugin-page-creator/src/derive-path.ts

```diff
@@ -34,7 +34,7 @@ export function derivePath(path: string, node: Record<string, any>): string {
     // 3.c  log error if the key does not exist on node
     if (nodeValue === undefined) {
       console.error(
```

Comment:
Let's move to reporter here too
```suggestion
      reporter.error(
```